orte/mca/ess/pmi/ess_pmi_module.c

```diff
@@ -203,7 +203,7 @@ static int rte_init(void)
     /* get the number of local peers - required for wireup of
      * shared memory BTL */
     OPAL_MODEX_RECV_VALUE(ret, OPAL_PMIX_LOCAL_SIZE,
-                          ORTE_PROC_MY_NAME, &u32ptr, OPAL_UINT32);
+                          &name, &u32ptr, OPAL_UINT32);
```

Comment:
yep, looks fine as well.
Currently I'm constantly worry that `name.rank` may change somewhere in between and I can't be sure that at certain point it still contains `rank == wildcard`.
Technically the same applies to `wildcard_rank` but here you understand that original developer ment a wildcard here and unlikely that he would change it somewhere in between.